aws/src/main/java/org/apache/iceberg/aws/glue/IcebergToGlueConverter.java

```diff
@@ -132,4 +138,61 @@ static void validateTableIdentifier(TableIdentifier tableIdentifier) {
     validateNamespace(tableIdentifier.namespace());
     validateTableName(tableIdentifier.name());
   }
+
+  static List<Column> toColumns(Schema schema) {
+    return schema.columns().stream()
+        .map(field -> Column.builder()
+            .name(field.name())
+            .type(toTypeString(field.type()))
+            .comment(field.doc())
+            .build())
+        .collect(Collectors.toList());
+  }
+
+  private static String toTypeString(Type type) {
```

Comment:
I haven't fully thought about the type conversion yet, just want to throw this here for discussion.

Reply:
This looks really similar to the conversion in `HiveSchemaUtil`. Is it possible to share that?